src/components/PlayerDash.jsx

```diff
@@ -27,6 +27,10 @@ export const PlayerDash = React.createClass({
     this.loadData(); // should update to bindAsObject/Array
   },
 
```

Comment:
@philals My version of PlayerDash still isn't using the firebase wrapper. This will need to be updated on your branch as well to use the wrapper version  `this.firebase.unload();`